odl/solvers/util/callback.py

```diff
@@ -315,6 +315,9 @@ def __init__(self, *args, **kwargs):
         ----------
         display_step : positive int, optional
             Number of iterations between plots. Default: 1
+        saveto : string, optional
```

Comment:
It is moved into kwargs, but the arguments listed here are the arguments which are not just directly piped into kwargs of show, which in turn I think pipes most of it into matplotlib.